MPF/ViewModels/MainViewModel.cs

```diff
@@ -516,13 +516,18 @@ public async void InitializeUIValues(bool removeEventHandlers, bool rescanDrives
                 await App.Instance.Dispatcher.InvokeAsync(DetermineSystemType);
             }
 
-            // Determine current media type, if possible
-            await App.Instance.Dispatcher.InvokeAsync(PopulateMediaType);
-            CacheCurrentDiscType();
-            SetCurrentDiscType();
+            // Are we just updating the output filename and path?
+            if (!skipTypeDetection)
+            {
+                // Determine current media type, if possible
+                await App.Instance.Dispatcher.InvokeAsync(PopulateMediaType);
+                CacheCurrentDiscType();
+                SetCurrentDiscType();
+            }
 
             // Set the initial environment and UI values
-            SetSupportedDriveSpeed();
+            if (!skipTypeDetection)
```

Comment:
This `if` can be merged into the proceeding one. Though at second look I don't fully understand why you're gating this off behind the `if` to begin with.

Reply:
Perhaps it doesn't need to be gated. I will make sure that keeping it doesn't reset the drive speed dropdown.

Reply:
I think it's reasonable if the speed changes back to the default. This can guard against too-fast or too-slow dumps if the last disc was particularly difficult.